eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2233,6 +2240,22 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
     context.system.eventStream.publish(NetworkFeePaid(self, remoteNodeId, channelId, tx, fee, desc))
   }
 
+  /**
+    * Returns the number of confirmations needed to safely handle the funding transaction,
+    * we make sure the cumulative block reward largely exceeds the channel size.
+    *
+    * @param fundingSatoshis funding amount of the channel
+    * @return number of confirmations needed
+    */
+  def minDepthForFunding(fundingSatoshis: Satoshi): Long = fundingSatoshis match {
```

Comment:
This static method should be in `Helpers.scala` right before `validateParamsFundee`.